assets/js/editor.js

```diff
@@ -129,6 +129,31 @@ $gp.editor = (
 					} else {
 						$gp.editor.save( target.parents( 'tr.editor' ).find( 'button.ok' ) );
 					}
+				} else if ( ( 13 === e.keyCode && e.ctrlKey ) || ( 66 === e.keyCode && e.shiftKey && e.ctrlKey ) ) { // Ctrl-Enter or Ctrl-Shift-B = Copy original.
```

Comment:
Shouldn't this just trigger a click event for `a.copy` so that the `copy()` method gets used?